packages/loader/container-loader/src/container.ts

```diff
@@ -1799,6 +1801,21 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         // are set. Global requests will still go directly to the loader
         const loader = new RelativeLoader(this.loader, () => this.containerUrl);
         const previousCodeDetails = this._context?.codeDetails;
+        const loadContainerCopyFn = async (
+            clientDetails?: IClientDetails,
+            fromSequenceNumber?: number,
+            summarizingClient?: boolean,
```

Comment:
loader layer shouldn't be aware of summarization

Reply:
i dugged a dig and apparently this arg just doesn't matter.  it gets used by `odspDriverUrlResolver` to set a `summarizer` property on an `IOdspResolvedUrl`, but nothing ever reads it and the summarizer decides to do it's job based entirely on other things.